src/Php73/bootstrap.php

```diff
@@ -17,6 +17,9 @@ function is_countable($var) { return is_array($var) || $var instanceof Countable
     }
 
     if (!function_exists('hrtime')) {
+        if (!class_exists(p\Php73::class)) {
+            require_once('Php73.php');
```

Comment:
I'd suggest removing the "if" and let `require_once` do its conditional job.
Also, this should be written as: require_once __DIR__.'/Php73.php';